bin/workflows/pycbc_create_offline_search_workflow

```diff
@@ -213,6 +212,49 @@ statfiles = []
 statfiles += wf.setup_trigger_fitting(workflow, insps, hdfbank,
                                       final_veto_file, final_veto_name)
 
+# setup the detector combination coincs for the filtering jobs
+full_insps = insps
+ctags = [tag, 'full']
+bg_files = []
+final_bg_files = []
+ifo_ids = {}
+
+# Get the ifo precedent list for the multiifo coinc jobs
```

Comment:
typo : 'precedence' not precedent

(I guess the description of what the precedence list *means* should be in the ini file, we'll deal with that later)